internal/common/utils.go

```diff
@@ -71,3 +75,9 @@ func CopyOutsideChroot(src, dest string) error {
 func GetStaterootPath(osname string) string {
 	return fmt.Sprintf("/ostree/deploy/%s", osname)
 }
+
+// FuncTimer check execution time
+func FuncTimer(now time.Time, name string, r logr.Logger) {
```

Comment:
nit: "start" would be a better name for the parameter than "now", since at the point where this is getting called, this time arg isn't actually "now", it's some point in the past. ie. "time elapsed since start"

Reply:
Updated